[BUG] Property 'table' does not exist on type 'ModelType<Document>'.

<!-- Not filling out ALL of the relevant fields in this issue will cause your issue to be closed -->

### Summary:
As per documentation, following code snippet should return the object used to create the table with AWS. But it fails to compile by TS.

### Code sample:

#### General
```
const User = dynamoose.model("User", {"id": Number, "name": String});

async function printTableRequest() {
    console.log(await User.table.create.request());
}
```


### Current output and behavior (including stack trace):

```
src/model.ts:79:33 - error TS2339: Property 'table' does not exist on type 'ModelType<Document>'.

79     console.log(await User.table.create.request());
```

### Expected output and behavior:




### Environment:

Operating System: Mac
Operating System Version: 10.15.6 (catalina)
Node.js version (`node -v`): v12.16.3
NPM version: (`npm -v`): 6.14.6
Dynamoose version: 2.3.0


### Other information (if applicable):




### Other:
- [x] I have read through the Dynamoose documentation before posting this issue
- [x] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [x] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [x] I have tested the code provided and am confident it doesn't work as intended
- [x] I have filled out all fields above
- [x] I am running the latest version of Dynamoose

--- a/test/types/Model.ts
+++ b/test/types/Model.ts
@@ -11,7 +11,9 @@ const shouldFailWithBooleanAsName = dynamoose.model(true);
 
 const shouldSucceedWithOnlyPassingInName = dynamoose.model("User");
 
-const model = dynamoose.model("User");
+const model = dynamoose.model("User", {"id": Number});
+
+const shouldPassTableCreateRequest = model.table.create.request();
 
 const shouldPassCreateWithNoReturnSetting = model.create({"id": 1}, {"overwrite": true});
 const shouldPassGetWithNoReturnSetting = model.get({"id": 1}, {"attributes": ["something"]});

